web/docs/how-does-tenzir-work/README.md

```diff
@@ -0,0 +1,66 @@
+# How Does Tenzir Work?
+
+Tenzir revolves around three primary components:
+
+- **Pipeline**: A defined sequence of operators responsible for loading,
+  parsing, transforming, and routing data. Pipelines are the core mechanism for
+  data processing.
+- **Node**: A running process that manages and executes pipelines.
+- **Platform**: A higher-level management layer that provides oversight and
+  control over multiple nodes.
+
+## Standalone and Managed Pipelines
+
+Tenzir offers flexibility in running pipelines through two main modes:
+
+1. **Standalone Execution**: Run directly from the command line using the
+   `tenzir` binary. This mode is ideal for quick, manual data transformations
+   akin to how one would use `jq`, but with broader data handling capabilities.
+2. **Managed Execution via Nodes**: For more sustained or automated data
+   processing, pipelines can be managed within a node. This is either launched
+   as a Docker container or by running the `tenzir-node` binary.
+
+![Pipelines Two Ways](user-journey-1.svg)
+
+The standalone option caters to interactive, ad-hoc tasks, while managed
+pipelines are better suited for continuous or scheduled dataflows. Notably,
+managed pipelines can handle both long-term and short-duration jobs.
+
+## Connecting Nodes to the Platform
+
+When deploying a node, it will automatically attempt to connect to the platform,
+providing a seamless way to manage and deploy pipelines through the platform’s
+web interface. However, using the platform is optional—pipelines can still be
+controlled directly via the node’s API in a more hands-on, CRUD-like approach.
```

Comment:
Maybe you want to add a sentence here that the platform can also run on-prem? There is a hint of it in the image below, but it's not too visible.